benchmarks/bench_lof.py

```diff
@@ -1,106 +0,0 @@
-"""
-============================
-LocalOutlierFactor benchmark
-============================
-
-A test of LocalOutlierFactor on classical anomaly detection datasets.
-
-Note that LocalOutlierFactor is not meant to predict on a test set and its
-performance is assessed in an outlier detection context:
-1. The model is trained on the whole dataset which is assumed to contain
-outliers.
-2. The ROC curve is computed on the same dataset using the knowledge of the
-labels.
-In this context there is no need to shuffle the dataset because the model
```

Comment:
please add this to the new example.

Reply:
Got it.

Reply:
This file should be removed. `bench_isolation_forest.py` should be kept as it is a novelty detection benchmark but `bench_lof.py` should be removed as it is replaced by the new example.